tools/cellxgene_census_builder/src/cellxgene_census_builder/release_manifest.py

```diff
@@ -25,11 +25,13 @@
 CensusVersionDescription = TypedDict(
     "CensusVersionDescription",
     {
-        "release_date": Optional[str],  # date of release, optional
+        "release_date": Optional[str],  # date of release (deprecated)
         "release_build": str,  # date of build
         "soma": CensusLocator,  # SOMA objects locator
         "h5ads": CensusLocator,  # source H5ADs locator
         "do_not_delete": Optional[bool],  # if set, prevents automated deletion
+        "flags": Optional[Dict[str, bool]],  # flags for the release
```

Comment:
since Builder is not manipulating the contents of "flags" (for now), I left this more loosely typed than what we have on the API side, which defines `ReleaseFlags` type, et. al.